[BE][cutlass backend] BE changes post cutlass_cppgen name change

henrylhtsang opened the pull request:
Differential Revision: D83809105

Handle reviews from https://github.com/pytorch/pytorch/pull/164159


cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @chenyang78 @kadeng @muchulee8 @amjames @chauhang @aakhundov @coconutruben

Skylion007 commented on torch/_inductor/codegen/cuda/cutlass_utils.py:
```diff
@@ -40,20 +40,17 @@ def move_cutlass_compiled_cache() -> None:
+        shutil.move(cutlass_cppgen.CACHE_FILE, os.path.join(cache_dir(), filename))
         log.debug("Moved CUTLASS compiled cache file to %s", cache_dir())
     except OSError as e:
         log.warning("Failed to move CUTLASS compiled cache file: %s", str(e))
```
```suggestion
        log.warning("Failed to move CUTLASS compiled cache file: %s", e)
```
'%s' does an implicit str cast anyway

Skylion007 commented on torch/_inductor/codegen/cuda/cutlass_utils.py:
```diff
@@ -422,7 +419,7 @@ def get_max_alignment(inductor_layout: Layout) -> int:
     offset = inductor_layout.offset
 
-    def is_static_int(number):
+    def is_static_int(number: object) -> bool:
```
```suggestion
    def is_static_int(number: object) -> TypeIs[int | sympy.Integer]:

henrylhtsang replied:
actually I don't feel like I understand why we should return TypeIs

henrylhtsang committed:
[BE][cutlass backend] BE changes post cutlass_cppgen name change (#164589)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/164589

Differential Revision: D83809105

henrylhtsang commented:
@pytorchbot merge

henrylhtsang commented:
@pytorchbot merge
